dashboard/utils_test.py

```diff
@@ -0,0 +1,396 @@
+        )
+
+        assert mmtrack.financial_aid_applied is True
+        assert mmtrack.financial_aid_status == fin_aid.status
+        assert mmtrack.financial_aid == fin_aid
+        assert mmtrack.financial_aid_min_price == 250
+        assert mmtrack.financial_aid_max_price == 1000
+
+    def test_course_price_mandatory(self):
+        """
+        Test that if financial aid is available for the program, at least one course price should be available.
+        """
+        program = ProgramFactory.create(live=True, financial_aid_availability=True)
+        TierProgramFactory.create(
+            program=program,
+            discount_amount=750,
+            current=True
+        )
+        with self.assertRaises(ImproperlyConfigured):
+            MMTrack(
+                user=self.user,
+                program=program,
+                enrollments=self.enrollments,
+                current_grades=self.current_grades,
+                certificates=self.certificates
+            )
+
+    def test_course_tier_mandatory(self):
+        """
+        Test that if financial aid is available for the program, at least one tier should be available.
+        """
+        program = ProgramFactory.create(live=True, financial_aid_availability=True)
+        crun_fa = CourseRunFactory.create(
+            program=program
+        )
+        CoursePriceFactory.create(
+            course_run=crun_fa,
+            is_valid=True,
+            price=1000
+        )
+        with self.assertRaises(ImproperlyConfigured):
+            MMTrack(
+                user=self.user,
+                program=program,
+                enrollments=self.enrollments,
+                current_grades=self.current_grades,
+                certificates=self.certificates
+            )
+
+    def test_is_course_in_program(self):
+        """
+        Test the _is_course_in_program method
+        """
+        # pylint: disable=protected-access
+        mmtrack = MMTrack(
+            user=self.user,
+            program=self.program,
+            enrollments=self.enrollments,
+            current_grades=self.current_grades,
+            certificates=self.certificates
+        )
+        for course_id in ["course-v1:edX+DemoX+Demo_Course", "course-v1:MITx+8.MechCX+2014_T1"]:
+            assert mmtrack._is_course_in_program(course_id) is True
+        assert mmtrack._is_course_in_program("course-v1:odl+FOO101+CR-FALL15") is False
+
+    def test_is_enrolled(self):
+        """
+        Tests for is_enrolled method
+        """
+        mmtrack = MMTrack(
+            user=self.user,
+            program=self.program,
+            enrollments=self.enrollments,
+            current_grades=self.current_grades,
+            certificates=self.certificates
+        )
+        for course_id in ["course-v1:edX+DemoX+Demo_Course", "course-v1:MITx+8.MechCX+2014_T1"]:
+            assert mmtrack.is_enrolled(course_id) is True
+
+        mmtrack_fa = MMTrack(
```

Comment:
Can you add a comment on what is being tested here, that wasn't tested in the previous assert?